languages/translations/en.json

```diff
@@ -41,7 +41,8 @@
                 "ox": "Keep left",
                 "xox": "Keep in the middle",
                 "oxo": "Keep left or right"
-            }
+            },
+            "ceremonialNamesToAvoid": ["freeway", "expressway", "highway", "fwy", "hwy", "expy"]
```

Comment:
“Fwy”, “hwy”, and “expy” shouldn’t occur in OpenStreetMap data; any occurrences would require edits to OSM.